### ISSUE TITLE
Multiple characters in one image

### ISSUE THREAD

**Issue by anujnayyar1** [HVision-NKU/StoryDiffusion#108]
Firstly thank you what an amazing project.

Is it possible to support multiple characters from multiple ref_images !

A cheeky extra ask - if this could be added to the replicate demo, it would be much appreciated.

**Comment by brentjohnston** [comment]
Yes, but there is some code that was added to temporarily disable two people, use chatgpt 4o to find it and you can simple remove it, it's only like a few lines.
